### PR TITLE
fix: remove remaining legacy import

### PR DESCRIPTION
**Pull request by gsontag**

### CODE REVIEW

File: projects/core/src/datetimepicker/datetimepicker-input.ts [hunk 1]
```diff
@@ -22,8 +22,8 @@ import {
   ValidatorFn,
   Validators,
 } from '@angular/forms';
-import { MAT_LEGACY_INPUT_VALUE_ACCESSOR as MAT_INPUT_VALUE_ACCESSOR } from '@angular/material/legacy-input';
```
Comment (bbortt) [inline projects/core/src/datetimepicker/datetimepicker-input.ts:25]: what about the constant `MAT_LEGACY_INPUT_VALUE_ACCESSOR `?

Comment (gsontag) [reply projects/core/src/datetimepicker/datetimepicker-input.ts:25]: Dawn, I miss one, I'll push fix soon

Comment (gsontag) [reply projects/core/src/datetimepicker/datetimepicker-input.ts:25]: In fact, I haven't forgot it, just the two imports have been reordered, but 
MAT_LEGACY_INPUT_VALUE_ACCESSOR became MAT_INPUT_VALUE_ACCESSORT
and MatLegacyFormField became MatFormField.

Comment (bbortt) [reply projects/core/src/datetimepicker/datetimepicker-input.ts:25]: hmm I see, sorry. I did misread that too :D

**Review by bbortt — APPROVED** [review]

**Pull request merged**